lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -193,6 +194,25 @@ class AlgoWorker {
       throw new Error('Failed to start algo order')
     }
   }
+
+  async generateAuthToken (url, apiKey, apiSecret) {
+    const rest = new RESTv2({
+      url,
+      apiKey,
+      apiSecret,
+      transform: true
+    })
+
+    const [authToken] = await rest.generateToken({
+      scope: 'api',
+      writePermission: true,
+      _cust_ip: null,
+      ttl: 5 * 60 * 60,
+      caps: ['f']
```

Comment:
why just for `funding` here ?

Reply:
also, since we are using api key and secret and it just occured to me.. idk how it comes into play while generating auth tokens using api key and secret but afaik, users give certain permissions to those api key.. will have to dig more into it .. but @robertkowalski might have some insights to this

Reply:
You're right, this was a copy-paste mistake. Now I've changed it to use account, orders, and wallets (a,o,w) capabilities, do you guys think this is enough to run the AO Host? Is there a place where I can check this?

Regard trying to generate a token with more capabilities than the API is allowed, I'll do some tests and bring some inputs.

(cc: @robertkowalski)

Reply:
If one tries to generate tokens with more capabilities it will throw the following error:

`[ 'error', null, 'ERR_TOKEN_CAPS_POLICY_INVALID' ]`

We probably should categorize with caps are necessary for the ao host to run completely and explicitly tell our consumers about which caps he should enable when managing his API credentials. 

WDYT? (cc: @robertkowalski)

Reply:
hello guys, if we generate auth token with specific caps via api key/secret then they should not exceed permissions specified while creating api key. Do we know in our case which caps we need, we could add some kind of note on UI to paste an api key with at least X capabilites

Reply:
@vigan-abd I'm not sure about the known list of caps, but for now, I'm using the following: `['a', 'o', 'w']` and it's working fine so far. We can evolve the list as we go.

Reply:
since hf mostly is used to make orders and open/close positions I think we should be good with current ones (account, orders, wallets). if we introduce also funding then we should add also `f` flag